path_static_roles.go

```diff
@@ -20,10 +21,24 @@ const (
 	staticRolePath = "static-role/"
 )
 
+// GenericNameWithForwardSlashRegex is a regex which requires a role name. The role name can
+// include any number of alphanumberic characters separated by forward slashes.
+func GenericNameWithForwardSlashRegex(name string) string {
+	return fmt.Sprintf("(/(?P<%s>\\w(([\\w-./]+)?\\w)?))", name)
+}
+
+// OptionalGenericNameWithForwardSlashListRegex is a regex for optionally including a
+// role path in list options. The role path can be used to list nested roles at
+// arbitrary depth.
+func OptionalGenericNameWithForwardSlashListRegex(name string) string {
+	// TODO(JM): List regex should support optional trailing slash
```

Comment:
I see this is still a TODO referred to in the tests. Will it be addressed in a follow up PR?

Reply:
Yes, I think so. I am deferring this until I get all of the API's updated. I wasn't able to figure out how to make the trailing slash optional but I think it is required as per the docs on the `Pattern` field in `framework.Path`